www/js/recommendations.js

```diff
@@ -27,13 +31,16 @@ angular.module('emission.main.recommendations',['emission.services', 'emission.p
       $scope.currentDisplay = document.getElementById("reviewModal" + id);
       $scope.currentDisplay.style.display = "block";
     };
-    $scope.getDistance = function(x, y) {
-      var xDif = ($scope.xCoord - x) * 55;
-      var yDif = ($scope.yCoord - y) * 69;
-      var x2 = Math.pow(xDif, 2);
-      var y2 = Math.pow(yDif, 2);
-      var dist = Math.sqrt(x2 + y2);
-      return Math.round(dist * 10) / 10 + " miles away";
-    }
+
+    //waiting on Vanessa integration 
+    $scope.getSuggestion = function() {
+      return new Promise(function(resolve, reject) {
+        var msgFiller = function(message) {
+          console.log("THIS IS THE SUGGESTION" + JSON.stringify(message));
```

Comment:
Not sure what this is doing. `msgFiller` is supposed to fill in the message to send *to* the server. Why would you send the suggestion to the server?